README.rst

```diff
@@ -257,6 +257,22 @@ Actions:
 * Update a deal - ``client.deals.update``
 * Delete a deal - ``client.deals.destroy``
 
+**A note about deal value**
+
+It is prefered to use decimal from string or directly string for deal values when creating or modifying a deal. This guarantees correct precision
+
+.. code:: python
+
+    deal.value = decimal("1000.99")
+    deal.value = "1000.00"
+
+You can also use floats and decimals from other tyoes, but we cannot guarantee correct precision when doing so.
```

Comment:
Maybe instead of "You can also use floats ..." let's say "You should not be using floats or decimal constructed from floats as it may result in precision loss."